publish-app-do/index.md

```diff
@@ -0,0 +1,458 @@
+  help                Help about any command
+  init                Initialize private validator, p2p, genesis, and application configuration files
+  keys                Manage your application's keys
+  migrate             Migrate genesis to a specified target version
+  query               Querying subcommands
+  start               Run the full node
+  status              Query remote node for status
+  tendermint          Tendermint subcommands
+  tx                  Transactions subcommands
+  unsafe-reset-all    Resets the blockchain database, removes address book files, and resets data/priv_validator_state.json to the genesis state
+  validate-genesis    validates the genesis file at the default location or at the location passed as an arg
+  version             Print the application binary version information
+
+Flags:
+  -h, --help                help for digitaloceand
+      --home string         directory for config and data (default "/home/appuser/.digitalocean")
+      --log_format string   The logging format (json|plain) (default "plain")
+      --log_level string    The logging level (trace|debug|info|warn|error|fatal|panic) (default "info")
+      --trace               print out full stack trace on errors
+
+Use "digitaloceand [command] --help" for more information about a command.
+```
+
+## Create a Systemd process
+
+The `starport serve` command helps installing, building, and configuring your app for a new blockchain. You received tokens on newly created accounts.
+
+This new app is useful only for development purposes and is not intended to be run for a long time in production. 
+
+For the app to run for a longer time, you need a `systemd` process.
+
+Create the log files and a `systemd` file to manage your process so that you can log off from the terminal and be confident the app keeps running even when the server restarts.
+
+```bash
+sudo mkdir -p /var/log/digitaloceand
+sudo touch /var/log/digitaloceand/digitaloceand.log
+sudo touch /var/log/digitaloceand/digitaloceand_error.log
+sudo touch /etc/systemd/system/digitaloceand.service
+```
+
+Use nano to edit the systemd service file:
+
+```bash
+sudo nano /etc/systemd/system/digitaloceand.service
+```
+
+Add your application details to the file:
+
+```bash
+[Unit]
+Description=digitalocean daemon
+After=network-online.target
+[Service]
+User=appuser
+ExecStart=/home/appuser/go/bin/digitaloceand start --home=/home/appuser/.digitalocean
+WorkingDirectory=/home/appuser/go/bin
+StandardOutput=file:/var/log/digitaloceand/digitaloceand.log
+StandardError=file:/var/log/digitaloceand/digitaloceand_error.log
+Restart=always
+RestartSec=3
+LimitNOFILE=4096
+[Install]
+WantedBy=multi-user.target
+```
+
+To close and save the file, press Ctrl+x. Then type `Y` and press Enter. 
+
+The next step is to enable the file so the app is available even after a system start:
+
+```bash
+sudo systemctl enable digitaloceand.service
+```
+
+Now start the process:
```

Comment:
```suggestion
Now start the systemd process:
```

Reply:
I started getting confused at this point, we're starting the systemd process?

Reply:
yes 👍